.github/workflows/build_and_test.yml

```diff
@@ -751,13 +752,16 @@ jobs:
         Rscript -e "install.packages(c('devtools', 'testthat', 'knitr', 'rmarkdown', 'markdown', 'e1071', 'roxygen2', 'ggplot2', 'mvtnorm', 'statmod'), repos='https://cloud.r-project.org/')"
         Rscript -e "devtools::install_version('pkgdown', version='2.0.1', repos='https://cloud.r-project.org')"
         Rscript -e "devtools::install_version('preferably', version='0.4', repos='https://cloud.r-project.org')"
+    # Should merge this step with 'Install Python linter dependencies' after SPARK 3.5 EOL.
```

Comment:
`Install Python linter dependencies` actually affects `Install dependencies for documentation generation` a lot, we should merge them in the future

Reply:
Why not merge all these requirements into the existing `dev/requirements.txt`?